modules/generator-primer-module/app/templates/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "0.1.0",
+  "name": "<%= module %>",
+  "description": "<%= description %>",
+  "homepage": "http://primercss.io/",
+  "author": "GitHub, Inc.",
+  "license": "MIT",
```

Comment:
Guessing you copied this over from my blueprint pr, since then we've added keys for primer `category` and `module_type`. Could this be prompts in the generator script too? Example:
```
"primer": {
    "category": "product",
    "module_type": "components"
  },
```